local-test-configuration/airseeker/airseeker.example.json

```diff
@@ -16,7 +16,8 @@
         "sanitizationSamplingWindow": 900,
         "sanitizationPercentile": 80,
         "scalingWindow": 120,
-        "maxScalingMultiplier": 2
+        "maxScalingMultiplier": 2,
+        "sanitizationMultiplier": 2
```

Comment:
Note, that for the actual values @bbenligiray mentioned using `50` as percentile and (I think) `3` for sanitization multiplier.